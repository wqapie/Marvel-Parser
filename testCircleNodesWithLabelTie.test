# Tests for a one-direction circle case with a single path, including testing successful loading
# (multiple children lexicographic ordering).

LoadGraph circleNodesWithLabelTie circleNodesWithLabelTie.tsv

ListChildren circleNodesWithLabelTie A
ListChildren circleNodesWithLabelTie B
ListChildren circleNodesWithLabelTie C

FindPath circleNodesWithLabelTie A C

FindPath circleNodesWithLabelTie B A

ListNodes circleNodesWithLabelTie