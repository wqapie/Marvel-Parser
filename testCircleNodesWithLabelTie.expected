# Tests for a one-direction circle case with a single path, including testing successful loading
# (multiple children lexicographic ordering).

loaded graph circleNodesWithLabelTie

the children of A in circleNodesWithLabelTie are: B(1)
the children of B in circleNodesWithLabelTie are: A(1) C(2)
the children of C in circleNodesWithLabelTie are: B(2)

path from A to C:
A to B via 1
B to C via 2

path from B to A:
B to A via 1

circleNodesWithLabelTie contains: A B C